src/css/eva.css

```diff
@@ -16,23 +16,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
This file contains many duplicated rules. I have pointed out some but there are more.